build/AzurePipelinesTemplates/WindowsAppSDK-RunTests-Steps.yml

```diff
@@ -0,0 +1,115 @@
+parameters:
+  buildPlatform: ''
+  buildConfiguration: ''
+  ImageName: ''
+  TaefSelect: '*'
+  BinaryCompatSwitch: ''
+  testLocale: ''
+steps:
+  # Download the BuildOutput from the Build stage(s).
+  # We only bring down the relevant content for this build config (Debug/Release) & platform, to save some space and time.
+  - task: DownloadBuildArtifacts@0
+    displayName: 'Download: BuildOutput'
+    inputs:
+      artifactName: 'BuildOutput'
+      downloadPath: $(Build.SourcesDirectory)\BuildOutput
+
+  # We only bring down the relevant content for this build config (Debug/Release) & platform, to save some space and time.
+  - task: DownloadBuildArtifacts@0
+    displayName: 'Download: Test dependencies'
+    inputs:
+      artifactName: 'packages'
+      downloadPath: $(Build.SourcesDirectory)
+
+    # We only bring down the relevant content for this build config (Debug/Release) & platform, to save some space and time.
+  - task: DownloadBuildArtifacts@0
+    displayName: 'Download: WinAppSDK Test Certificate'
+    inputs:
+      artifactName: 'TestCert'
+      downloadPath: $(Build.SourcesDirectory)\BuildOutput
+
+  - task: powerShell@2
+    displayName: 'Enable developer mode'
+    inputs:
+      targetType: 'inline'
+      script: |
+            $RegistryKeyPath = "HKLM:\SOFTWARE\Microsoft\Windows\CurrentVersion\AppModelUnlock"
```

Comment:
What's this needed for? Just curious

Reply:
This is used for putting the machine in Developer mode.